life_expectancy/cleaning.py

```diff
@@ -1,32 +1,73 @@
 import argparse
+from abc import ABC, abstractmethod
 import pandas as pd
 from life_expectancy.i_o import load_data, save_data
+from life_expectancy.countries import Country
 
-def clean_data(df):
+class FileCleaner(ABC):
+    """
+    Abstract class for reading different types of files
+    """
+    ext=''
+    @abstractmethod
+    def clean_data(self, df: pd.DataFrame):
```

Comment:
```suggestion
    def clean_data(self, df: pd.DataFrame) -> pd.DataFrame:
```
Mister Miyagi says, type hint goes in, type hint comes out.

Reply:
Addresed